packages/client/hmi-client/src/workflow/ops/model-config/model-config-operation.ts

```diff
@@ -1,9 +1,14 @@
 import { Operation, WorkflowOperationTypes } from '@/types/workflow';
+import { Initial, ModelParameter } from '@/types/Types';
 
 export const name = 'ModelConfigOperation';
 
 export interface ModelConfigOperationState {
 	modelId: string | null;
+	configName: string;
+	configDescription: string;
+	configInitials?: Initial[];
+	configParams?: ModelParameter[];
```

Comment:
Remove the `config` prefix form these names, they're already wrapped around `ModelConfigOperationState`. Reflect this in the ref names in `tera-model-config.vue`